templates/include/telemetry/syncItem.1.schema.json

```diff
@@ -0,0 +1,99 @@
+{
+  "type": "object",
+  "properties": {
+    "didLogin": { "type": "boolean" },
+    "when": { "type": "integer" },
+    "devices": {
+      "type": "array",
+      "items": {
+        "additionalProperties": false,
+        "type": "object",
+        "properties": {
+          "id": { "type": "string", "pattern": "^[0-9a-f]{64}$" },
+          "os": { "type": ["string", "null"] },
+          "version": { "type": "string" }
+        }
+      }
+    },
+    "status": {
+      "type": "object",
+      "anyOf": [
```

Comment:
Will this type of use of `anyOf` cause this section to be ignored?

Reply:
Yes, but this section is limited to setting the field to required, so `sync` and `service` will end up being nullable fields. This is fine since all fields are going to be treated as nullable with the upcoming change to MSG.